### PR TITLE
CEDS-1863 Confirmation Page

### PR DESCRIPTION
**Pull request by ed0906**

### CODE REVIEW

File: app/storage/FlashKeys.scala [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package storage
-
-object FlashKeys {
```
Comment (PatrykRudnicki) [inline app/storage/FlashKeys.scala]: Why you removed FlashKeys? You're still using it. I can't find new place for it

File: app/controllers/consolidations/DisassociateDucrConfirmationController.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import play.api.mvc._
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.html.disassociate_ducr_confirmation
+
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class DisassociateDucrConfirmationController @Inject()(authenticate: AuthenticatedAction,
+                                                       getJourney: JourneyRefiner,
+                                                       mcc: MessagesControllerComponents,
```
Comment (PatrykRudnicki) [inline app/controllers/consolidations/DisassociateDucrConfirmationController.scala]: Please use formatter

File: app/storage/FlashKeys.scala (hunk 1 above)
Comment (ed0906) [reply app/storage/FlashKeys.scala]: Theres one already

File: app/controllers/consolidations/DisassociateDucrConfirmationController.scala (hunk 2 above)
Comment (ed0906) [reply app/controllers/consolidations/DisassociateDucrConfirmationController.scala]: Done

**Pull request merged**